apps/docs/src/app/core/component-docs/text/examples/text-hyphenation.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'text-hyphenation',
+    templateUrl: './text-hyphenation.component.html',
+    styleUrls: ['./text-hyphenation.component.scss']
+})
+export class TextHyphenationComponent {
+    text = `Lorem ipsum dolor sit amet, con\u00ADsectetur adip\u00ADiscing el\u00ADit, sed do eiusmod tempor incididunt ut labore et do\u00ADlore magna aliqua. Ut enim ad mi\u00ADnim veniam, quis nostrud exer\u00ADcitation ullamco laboris nisi ut aliquip ex ea commodo conse\u00ADquat. Duis aute irure dolor in repre\u00ADhenderit in vo\u00ADluptate velit esse cillum dolore eu fu\u00ADgiat nulla pari\u00ADatur. Excepteur sint occaecat cupidatat non pro\u00ADident, sunt in culpa qui officia deserunt mollit anim id est la\u00ADbo\u00ADrum.`;
```

Comment:
Why do we have this `\u00AD` in the markup? Is it mendatory to have hyphenation?

Reply:
I suppose that it should be driven be css itself

Reply:
The main goal of this is to create clear examples, It was done by example of [MDN examples](https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens). Now I think, maybe it will be a good thing to add an example for the "manual" value

Reply:
Yeah, it would be good to have one example showing how browser does hyphenation by itself and add one more with a manual hyphenation rules.

Reply:
Added new example and a few extra changes